### ISSUE TITLE
[Feature] Support historical results on OSD Visualizations

### ISSUE THREAD

**Issue by ohltyler** [opensearch-project/anomaly-detection-dashboards-plugin#803]
Currently, the results fetched when triggered from OSD Visualizations are limited to real-time results (see [here](https://github.com/opensearch-project/anomaly-detection-dashboards-plugin/blob/main/public/expressions/helpers.ts#L44)). Ideally, users have the flexibility to also include historical results to be overlaid to provide more insights to users, particularly if they have large amounts of historical data and historical anomaly results.

**Comment by ohltyler** [comment]
Tracking issue to update documentation: https://github.com/opensearch-project/documentation-website/issues/7647

**Comment by dblock** [comment]
@ohltyler was this done?

[Catch All Triage - [1](https://github.com/dblock), [2](https://github.com/krisfreedain), [3](https://github.com/andrross), [4](https://github.com/reta)]

**Comment by ohltyler** [comment]
No, this is not done. The documentation has just been updated to reflect that this proposed feature (historical results) is not supported explicitly.
